conda/cli/helpers.py

```diff
@@ -205,6 +205,15 @@ def add_output_and_prompt_options(p: ArgumentParser) -> _ArgumentGroup:
     return output_and_prompt_options
 
 
+def add_parser_frozen_env(p: ArgumentParser):
+    p.add_argument(
+        "--ignore-frozen-env",
```

Comment:
This sounds like we are ignoring the env itself, not the protection:

```suggestion
        "--ignore-frozen-env-protection",
```

```suggestion
        "--override-frozen-env",
```

```suggestion
        "--override-frozen",
```

Reply:
After reading through everything I think I prefer `--override-frozen`

Reply:
I was going at it as in `--ignore-env-is-frozen`. Is that better?

We also have `--freeze-installed-packages`, so that's why I feel it's better to mention `env` explicitly in the flag name, for clarity.

Reply:
> We could use --break-base-environment as an alternative name

We can freeze any environment, not just base, so maybe we can use `--break-environment`?

Reply:
`--override-frozen`/`--override-frozen-env` makes more sense to me and wont require reading the docs to understand what is happening.

I prefer we stick to existing conda lingo. I consider `--override-channels` to have a very clear meaning in `conda` (or do I use that more than the average user?) so using the same terminology here makes it easier to understand. 🤷🏼‍♂️

Whatever is decided I think the argument needs to explicitly reference "frozen", forcing the user to have to learn two terms ("frozen env" and "break env") for a pretty simple feature is overkill.